### ISSUE TITLE
[Feature request] Alert or vibration when double tapping to copy

### ISSUE THREAD

**Issue by FrancoisLunick** [venkatasg/DeTeXt#5]
**Description:**
The app works great! However, I would like to request a feature enhancement. Currently, when double tapping to copy Unicode points or the character itself, there is no indication that the action has been successful. No sound, vibration, or alert is provided. This makes it difficult for users to know if their action was completed successfully.

**Details:**

	•	iOS Version: 18.1 beta
	•	Device: iPhone 14 Pro Max

I suggest adding a feedback mechanism, such as:

	1.	A vibration or haptic feedback.
	2.	A sound alert.
	3.	A visual confirmation (e.g., a small pop-up notification).

**Comment by venkatasg** [comment]
Good idea, I think I had something similar - a visual pop-up, but will also include a haptic feedback.

Currently aiming to have this (and other features) ready for iOS 18 release in September.

**Comment by ChengxianJin** [comment]
I second this! Could you also add some kind of visual pop-up (and/or Magic Trackpad haptic feedback) on macOS?

**Comment by venkatasg** [comment]
I don't know if it's possible to add Magic Trackpad feedback but I'll look into it. The macOS app is based on the iPad app so pop-ups should carry over.
